projects/ngrx.io/README.md

```diff
@@ -52,6 +52,19 @@ Running `yarn start` (even when explicitly targeting production mode) does not s
 ServiceWorker. If you want to test the ServiceWorker locally, you can use `yarn build` and then
 serve the files in `dist/` with `yarn http-server dist -p 4200`.
 
+## Running on Docker
+
+The ngrx.io sample project can run as a docker container. 
+In order to run ngrx.io on docker, use the following commands (**run commands from the platform folder**):
```

Comment:
> **run commands from the platform folder**

Should we move the docker files to the ngrx.io folder?

Reply:
> > **run commands from the platform folder**
> 
> Should we move the docker files to the ngrx.io folder?

At first, the docker files intended to be at the ngrx.io folder,
but when I tried doing it the `yarn setup` failed.
If it's critical I can look more into it (might take some time),
although I think it could be fine if the docker files would stay the way they are 😊